framework/graphics/vulkan_util.cpp

```diff
@@ -40,5 +40,36 @@ void ReleaseLoader(util::platform::LibraryHandle loader_handle)
     }
 }
 
+VkQueue GetDeviceQueue(const encode::VulkanDeviceTable* device_table,
+                       const decode::DeviceInfo*        device_info,
```

Comment:
Could we remove the dependency in `vulkan_util` on `decode` if this parameter was just the handle and the creation flags was passed in?

Reply:
Moved this function to `decoder_util.h`.

Reply:
My opinion is that `vulkan_util` is the right place for this if instead of taking `decode::DeviceInfo` it could just take a vkDevice and creation flags so that it didn't need to use `decode` namespace.

Reply:
Fixed it.

Reply:
Undo it.